pkg/kubelet/dockertools/manager.go

```diff
@@ -579,7 +579,7 @@ func (dm *DockerManager) runContainer(
 		}
 	}
 	memoryLimit := container.Resources.Limits.Memory().Value()
-	cpuShares := milliCPUToShares(container.Resources.Limits.Cpu().MilliValue())
+	cpuShares := milliCPUToShares(container.Resources.Requests.Cpu().MilliValue())
```

Comment:
Yes and no. It now treats request as it treated limit before. This goes towards my other note that this is a breaking change.

However, as part of this QoS effort (not in this PR though). Limit is implemented via hard-capping such that you will be throttled when you reach your limit.